packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -911,4 +911,20 @@ export class MirrorNodeClient {
     public getMirrorNodeWeb3Instance(){
         return this.web3Client;
     }
+
+    /**
+     * This method is intended to be used in cases when the default axios-retry settings do not provide
+     * enough time for the expected data to be propagated to the Mirror node.
+     * It provides a way to have an extended retry logic only in specific places
+     */
+    public async repeatedRequest(methodName: string, args: any[], repeatCount: number, requestId?: string) {
+        let result;
+        for (let i = 0; i < repeatCount; i++) {
+            result = await this[methodName](...args, requestId);
```

Comment:
Q: shouldn't there be some kind of backoff before retry?
Should you be waiting `MIRROR_NODE_RETRY_DELAY` ms?

Reply:
yeah, I thought the same.

Reply:
Added backoff